share/lib/python/neuron/rxdtests/tests/3d/include_flux3d.py

```diff
@@ -0,0 +1,43 @@
+from neuron import h, rxd
```

Comment:
A header should explain what this test is doing

Reply:
This and many other tests from the `share/lib/python/neuron/rxdtests/tests` directory will be in a subsequent PR implemented as pytest tests and should be documented there.